backend/src/hatchling/builders/wheel.py

```diff
@@ -37,6 +37,20 @@
 TIME_TUPLE = Tuple[int, int, int, int, int, int]
 
 
+WHEEL_NO_FILES_ERROR_MESSAGE_TEMPLATE = ("""
```

Comment:
Let's move this back down to an in-line message, I try my best to have nothing happen at import time